references/classification/train_quantization.py

```diff
@@ -65,7 +66,8 @@ def main(args):
                                                        step_size=args.lr_step_size,
                                                        gamma=args.lr_gamma)
 
-    model.to(device)
+    if args.distributed and args.sync_bn:
+        model = torch.nn.SyncBatchNorm.convert_sync_batchnorm(model)
```

Comment:
At some point in time, converting to SyncBatchNorm would (erroneously) create some tensors on the CPU, despite the model being on the GPU. I suppose this has been fixed? Otherwise we would need to call `model.to(device)` again after `SyncBatchNorm`.

Now that I think a bit more about this, looks like finetuning was wrong because of the location of the `to(device)` before, because it was done after creating the optimizer?

Reply:
yep, the SyncBatchNorm device affinity was fixed in https://github.com/pytorch/pytorch/pull/38729 and this diff assumes the fix is present in `torch`.

https://github.com/pytorch/vision/pull/2230 (by me) originally moved the `to(device)` down to unbreak the tutorial, because at that time device affinity was broken with QAT.  This PR moves it back to the original place, since the bugs are fixed.